lnrpc/routerrpc/forward_interceptor.go

```diff
@@ -100,6 +100,11 @@ func (r *forwardInterceptor) onIntercept(
 func (r *forwardInterceptor) resolveFromClient(
 	in *ForwardHtlcInterceptResponse) error {
 
+	if in == nil || in.IncomingCircuitKey == nil {
```

Comment:
I don't think `in` is ever expected to be `nil`. So can just do the second check